variables.tf

```
@@ -58,3 +58,40 @@ variable "session_affinity" {
   default     = "NONE"
 }
 
+variable "disable_health_check" {
+  type        = bool
+  description = "Disables the health check on the target pool."
+  default     = false
+}
+
+variable "health_check" {
+  description = "Health check to determine whether instances are responsive and able to do work"
+  type = object({
+    check_interval_sec  = number
+    healthy_threshold   = number
+    timeout_sec         = number
+    unhealthy_threshold = number
+    port                = number
+    request_path        = string
+    host                = string
+  })
+  default = {
+    check_interval_sec  = null
+    healthy_threshold   = null
+    timeout_sec         = null
+    unhealthy_threshold = null
+    port                = null
+    request_path        = null
+    host                = null
+  }
+}
+
+variable "ip_address" {
+  description = "IP address of the external load balancer, if empty one will be assigned. Default is empty."
```

We don't need to describe defaults unless behaviour is complex; defaults will be automatically rendered.

```suggestion
  description = "IP address of the external load balancer, if empty one will be assigned."
```